tests/testdata/version.txt

```diff
@@ -1 +1 @@
-938124458cdfdcd1b7b6bf14762ef606447c3c89
+ed2bb75d109c579ad9b1e8ad1efe089421326028
```

Comment:
Nit: You can put the version tag here. The `download_releases.sh` script does that for you. Will check that it works.

Reply:
The `download_releases.sh` works. I run it like this:

`$ GITHUB_API_TOKEN="$BBN_GITHUB_API_TOKEN" ./download_releases.sh v0.7.0-rc.0`

Where `BBN_GITHUB_API_TOKEN` contains a token with access to the `babylon-contract` repo.